docs/user-guide/configuration/coverage.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: main
+title: Coverage
+category: User Guide
+menu: menu
+toc:
+    - title: Coverage
+      url: "#coverage"
+---
+# Coverage
+
+Builds can upload coverage data after they are run.
+We currently support [SonarQube](https://github.com/screwdriver-cd/coverage-sonar) for coverage bookends.
+
+## SonarQube
+
+To use SonarQube, add a `sonar-project.properties` file in the root of your source code and add configurations there.
+
+Example `sonar-project.properties` file:
+```
+sonar.sources=lib
+sonar.javascript.lcov.reportPath=artifacts/coverage/lcov.info
```

Comment:
We should say something that the reportPath property is different depends on language. They should check this doc to figure out the right syntax: https://docs.sonarqube.org/display/PLUG